src/sentry/models/event.py

```diff
@@ -42,6 +43,9 @@ class Event(Model):
 
     objects = BaseManager()
 
+    # Key to use when ordering a list of events manually
+    ordering_key = attrgetter('datetime', 'id')
```

Comment:
its pretty obscure to have on the model (i also hate adding random things to the model) but I dont have a better suggestion